From a2f2163122fa521143514fdb5c49f7bb89e0c12d Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Mika=20Bostr=C3=B6m?= <bostik@odin.bostik.iki.fi>
Date: Tue, 22 May 2012 06:55:50 +0300
Subject: [PATCH] Fix modular build

Remove yet another DESTDIR abuse. These are likely artifacts from
developer systems where (some) system paths are writable.

These really need to go through the install step.
---
 src/plugins/platforms/wayland/wayland.pro |    2 --
 1 file changed, 2 deletions(-)

diff --git a/src/plugins/platforms/wayland/wayland.pro b/src/plugins/platforms/wayland/wayland.pro
index 73909bf..1cee395 100644
--- a/src/plugins/platforms/wayland/wayland.pro
+++ b/src/plugins/platforms/wayland/wayland.pro
@@ -4,8 +4,6 @@ load(qt_module)
 
 CONFIG += qpa/genericunixfontdatabase
 
-DESTDIR = $$QT.gui.plugins/platforms
-
 QT += core-private gui-private platformsupport-private
 
 SOURCES =   main.cpp \
-- 
1.7.10

